apps/api/src/beneficiary/beneficiary.controller.spec.ts

```diff
@@ -13,9 +14,15 @@ describe('BeneficiaryController', () => {
     }).compile()
 
     controller = module.get<BeneficiaryController>(BeneficiaryController)
+    // service = module.get<BeneficiaryService>(BeneficiaryService)
   })
 
   it('should be defined', () => {
     expect(controller).toBeDefined()
   })
+
+  // it('should return list of beneficiaries', () => {
+  //   const result = []
+  //   jest.spyOn(service, 'findAll').mockImplementation(() => result)
+  // })
```

Comment:
We don't need this commented code in the repo

Please either make it real or remove it